tests/base.py

```diff
@@ -7,10 +7,14 @@
 
 
 class TestClient(LTTestClient[Litestar]):
+    app: Litestar
```

Comment:
I don't understand, where is this initialized?

Reply:
This is so that you can use test_client.app without any typing errors...